### PR TITLE
feat(metrics): Extract more metrics from spans

### PR DESCRIPTION
**Pull request by iker-barriocanal**
Follow-up to https://github.com/getsentry/relay/pull/2069.

Adds the remaining metrics with their tags.

#skip-changelog

### CODE REVIEW

File: relay-server/src/metrics_extraction/transactions/mod.rs [hunk 1]
```diff
@@ -411,29 +413,104 @@ fn extract_span_metrics(
...
+                    span_tags.insert("module".to_owned(), module.to_owned());
+                }
+
+                let action = match span_module {
+                    Some("http") => span
+                        .data
+                        .value()
+                        // TODO(iker): some SDKs extract this as method
+                        .and_then(|v| v.get("http.method"))
+                        .and_then(|db_op| db_op.as_str()),
```
Comment (jjbayer) [inline relay-server/src/metrics_extraction/transactions/mod.rs]:
```suggestion
                        .and_then(|method| method.as_str()),
```

File: relay-server/src/metrics_extraction/transactions/mod.rs [hunk 2]
```diff
@@ -411,29 +413,104 @@ fn extract_span_metrics(
...
+                    _ => None,
+                };
+
+                if let Some(act) = action {
+                    span_tags.insert("action".to_owned(), act.to_owned());
+                }
+
+                // TBD product requirements, adding it as a placeholder
+                span_tags.insert("domain".to_owned(), "todo".to_owned());
```
Comment (jjbayer) [inline relay-server/src/metrics_extraction/transactions/mod.rs]: Does it even make sense to set the tag if we do not know the value yet?

**Review by jjbayer — APPROVED** [review]
LGTM

File: relay-server/src/metrics_extraction/transactions/mod.rs (hunk 2 above)
Comment (iker-barriocanal) [reply relay-server/src/metrics_extraction/transactions/mod.rs]: I discussed this with Jan, and he suggested adding it as a placeholder. I don't think it really helps, so I removed it; we could apply the same reasoning for the other unimplemented tags I assume.

**Comment by TBS1996** [comment]
would it make sense to also make a `enum SpanMetric` in the way we do with `TrasnactionMetric` and `SessionMetric`? https://github.com/getsentry/relay/blob/dda830c29fbfd9d5cf79efcad59dfd68f7ce1bfd/relay-server/src/metrics_extraction/transactions/types.rs#L13

**Comment by iker-barriocanal** [comment]
@TBS1996 Yes, it does. That is a follow-up step after we're no longer a blocker.

**Pull request merged**